src/test.ts

```diff
@@ -3,5 +3,12 @@ import * as jsdom from 'jsdom';
 const window = new jsdom.JSDOM('<main></main>').window;
 global.document = window.document;
 global.window = window;
```

Comment:
type error:

```
Error: src/test.ts(5,1): error TS2322: Type 'DOMWindow' is not assignable to type 'Window & typeof globalThis'.
  Type 'DOMWindow' is not assignable to type 'Window'.
    Types of property 'self' are incompatible.
      Type 'DOMWindow' is not assignable to type 'Window & typeof globalThis'.
        Type 'DOMWindow' is missing the following properties from type 'typeof globalThis': RTCStatsReport, AnalyserNode, Animation, AnimationEffect, and 489 more.
```

Reply:
I'll open a separate PR to fix these. Didn't realize they were build breaking.

Reply:
#20